### PR TITLE
feat(market-launch-notif): Add notification for market launch

### PR DESCRIPTION
**Pull request by jaredvu**
Screen Shot 2024-08-06 at 3 26 36 PM

Screen Shot 2024-08-06 at 3 27 27 PM

Add notification  and localized copy for feature launch.

---

## Views

* `<MarketsDropdown>`
  * Localize copy found on MarketsDropdown
  * add new `AmericanFlag2.png` and use as the `<TriggerContainer>` background
  * Update styling from using css selectors `first-of-type/last-of-type` to `tw` prop

* `<Markets>`
  * localize banner copy
  * Update banner gradient color

* `<TradeFormInfoMessages>`
  * Move testFlags check from `<TradeForm>` into this file.
  * localize copy

* `<MarketLaunchTrumpwinNotification>`
  * Add notification content  

## Constants/Types

* `constants/markets`
  * add stringKey for `PREDICTION_MARKET`
  * export `PREDICTION_MARKET` map to reduce manual input/change of market name

* `constants/notifications`
  * Add new `MarketLaunchNotificationIds` 

## Hooks

* `hooks/useNotificationTypes`
  * clean-up expired notifications
  * Add MarketLaunchNotification

* `hooks/useURLConfigs`
  * Expose `predictionMarketLearnMore` as well as add to `configs/v1/env.json`

## Packages

* `v4-localization`
  * updated: v1.1.167 -> v1.1.169

---

Testing:
Three items need to be done to test
- Add `Prediction Market` tag to `TRUMP-USD` to simulate having the `TRUMPWIN-USD` market.
- Add `?prediction=true` as a query param
- In `constants/markets.ts` update the key `TRUMPWIN` value `TRUMPWIN-USD` -> `TRUMP-USD`

### CODE REVIEW

File: src/views/notifications/MarketLaunchTrumpwinNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      slotAction={
+        <$Link to={`${AppRoute.Trade}/${PREDICTION_MARKET.TRUMPWIN}`}>
+          {stringGetter({ key: STRING_KEYS.TRADE_NOW })} →
+        </$Link>
+      }
+    />
+  );
+};
+
+const $Market = styled.span`
```
Comment (tyleroooo) [inline src/views/notifications/MarketLaunchTrumpwinNotification.tsx:38]: Might as well tailwind these! 
`text-color-text-1` and `text-color-accent visited:text-color-accent`

File: src/views/MarketsDropdown.tsx [hunk 2]
```diff
@@ -314,29 +333,6 @@ const $TriggerContainer = styled.div<{ $isOpen: boolean }>`
     css`
       --marketsDropdown-width: var(--marketsDropdown-openWidth);
     `}
-
-  > :first-child {
```
Comment (tyleroooo) [inline src/views/MarketsDropdown.tsx:318]: oh nice, inline tailwind looks much better here

File: src/views/MarketsDropdown.tsx [hunk 3]
```diff
@@ -244,31 +254,39 @@ export const MarketsDropdown = memo(
...
-          </$TriggerContainer>
+          <>
+            {triggerBackground}
+            <$TriggerContainer $isOpen={isOpen}>
+              {isOpen ? (
+                <h2 tw="text-color-text-2 font-medium-medium">
+                  {stringGetter({ key: STRING_KEYS.SELECT_MARKET })}
+                </h2>
+              ) : (
+                <div tw="spacedRow gap-[0.625rem]">
```
Comment (tyleroooo) [inline src/views/MarketsDropdown.tsx:273]: don't need brackets, just gap-0.625 should work

File: src/views/notifications/MarketLaunchTrumpwinNotification.tsx (hunk 1 above)
Comment (jaredvu) [reply src/views/notifications/MarketLaunchTrumpwinNotification.tsx:38]: YEEEEEE!!!!

Was waiting for that merge

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**